scripts/pipelines/acquia-pipelines.yml

```diff
@@ -16,6 +16,9 @@ events:
               - composer validate --no-check-all --ansi
               - composer install --ansi
               - source ${BLT_DIR}/scripts/pipelines/setup_env
+              - nvm install v6.11.0
```

Comment:
i would suggest using the 'lts/boron' name in case there are any incremental patch or minor updates, e.g., 

`nvm install lts/boron`